test/module/shared_model/cryptography/blob_test.cpp

```diff
@@ -15,19 +15,37 @@
  * limitations under the License.
  */
 
-#include <gtest/gtest.h>
 #include "cryptography/blob.hpp"
+#include <gtest/gtest.h>
 
 using namespace shared_model::crypto;
 
+auto data = "Hello World";
```

Comment:
I think it would be great to use some string with "bad" characters inside (like '\0') because it was a reason we introduced this pr.